src/index.ts

```diff
@@ -283,6 +292,19 @@ class Monika extends Command {
     }
   }
 
+  async openConfigGenerator(flags: any): Promise<boolean> {
+    const openPage = flags['create-config'] && !flags.har && !flags.postman
+
+    if (openPage) {
+      log.info(
+        'Opening Monika Configuration Generator in your default browser...'
+      )
+      await open('https://hyperjumptech.github.io/monika-config-generator/')
```

Comment:
cosmetic comment, feel free to ignore/resolve immediately: can we put `https://hyperjumptech.github.io/monika-config-generator/` inside constant variable?

Reply:
since monika is to be run on client's engine this intended to be hardcoded.